.gitignore

```diff
@@ -5,3 +5,4 @@
 /docs/Manifest.toml
 /docs/build/
 .vscode
+/examples/ising-*.jl
```

Comment:
I would prefer if you move ising-mwe.jl out of your local repo clone so we don't need to have this line